docs/fundamentals/platform.rst

```diff
@@ -1,2 +1,279 @@
+galadriel
+
+>>> print(platform.chip)
+Chip with 2 qubits and 6 ports:
+* Port drive_line_q0 (drive): ----|qubit_0|----
+* Port drive_line_q1 (drive): ----|qubit_1|----
+* Port flux_line_q0 (flux): ----|qubit_0|----
+* Port flux_line_q1 (flux): ----|qubit_1|----
+* Port feedline_input (feedline_input): ----|resonator_q0|--|resonator_q1|----
+* Port feedline_output (feedline_output): ----|resonator_q0|--|resonator_q1|----
+
+>>> print(platform.buses)
+Bus feedline_bus:  -----|QRM1|--|rs_1|------|resonator_q0|------|resonator_q1|----
+Bus drive_line_q0_bus:  -----|QCM-RF1|------|qubit_0|----
+Bus flux_line_q0_bus:  -----|QCM1|------|qubit_0|----
+Bus drive_line_q1_bus:  -----|QCM-RF1|------|qubit_1|----
+Bus flux_line_q1_bus:  -----|QCM1|------|qubit_1|----
+
+which displays the connections between the buses, instruments and elements of the chip.
+
+|
+
+Connecting and setting up the instruments with Platform:
+---------------------------------------------------------
+
+After building the platform, you need to connect to the instruments, set all the parameters defined in the runcard, and turn on the sources outputs using the following methods:
+
+.. code-block:: python
+
+    platform.connect()
+    # Connects to all the instruments and blocks the connection for other users.
+    # You must be connected to proceed with the following steps.
+
+    platform.initial_setup()
+    # Sets the values of the runcard (serialized platform) to the connected instruments.
+    # You might want to skip this step if you think no parameters have been modified since last time, but we recommend doing it anyway.
+
+    platform.turn_on_instruments()
+    # Turns on the signal output for the generator instruments (RF, voltage sources and current sources).
+    # This does not actually turn on the instruments of the laboratory, it only opens the signal output generation of the sources.
+    # You might want to skip this step if the instruments outputs are already open, but again, we recommend doing it anyway.
+
+.. note::
+
+    To connect, your computer must be in the same network of the instruments specified in the runcard (with their IP's addresses).
+
+|
+
+Executing a circuit with Platform:
+-----------------------------------
+The Platform offers the capability to execute circuits defined with `Qibo <https://qibo.science/>`_, an open-source middleware for quantum computing.
+
+To execute a circuit, you first need to build, connect, and set up the platform as shown in the above examples. Then, define your
+Qibo circuit, for example, a pi pulse and a measurement gate on qubit ``q`` (``int``):
+
+.. code-block:: python3
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    circuit = Circuit(q+1)
+    circuit.add(gates.X(q))
+    circuit.add(gates.M(q))
+
+And you are ready to execute the circuit with the platform:
+
+>>> result = platform.execute(program=circuit, num_avg=1000, repetition_duration=6000)
+>>> result.array
```

Comment:
perhaps `result.array` is gonna be something too big to display so you can instead run
`result = platform.execute(program=circuit, num_avg=1, repetition_duration=6000, num_bins=1000)`
result.counts to get the dictionary of counts for 0 and 1, which will be more compact (and people comming form qiskit or other libaries will be probably more familiar with)

Reply:
Yess, I'll add counts and probabilities! I had that one in mind already actually, thanks!